contracts/SplitMain.sol

```diff
@@ -230,6 +240,17 @@ contract SplitMain is ISplitMain {
 
   constructor() {
     walletImplementation = address(new SplitWallet());
+
+    SPLIT_DEPLOYER = tx.origin;
+    // configure gas fees as claimable
+    BLAST_YIELD.configureClaimableGas();
+    // configure eth yield as claimable
+    BLAST_YIELD.configureClaimableYield();
+    // set governor to deployer
+    BLAST_YIELD.configureGovernor(tx.origin);
```

Comment:
does tx.origin make more sense than msg.sender here?

Reply:
yeah, can be msg.sender since we just use normal create flow, changing this